oluwajuwon/Fast-Food-Fast
# Chore connect db #160914128

### What does this pull request do?
This pull request connects the already created endpoints to a PostgreSQL database

### Description of the Task to be completed?
The web app (Fast-food-fast) shouldn't be working with dummy data, it should be using a database to persist data and perform CRUD operations

### How should this be manually tested?
Users can test it by:
 - Downlaoding and Installing Postman from https://www.getpostman.com/
 - Send requests to the API endpoints through the following
   - Index route -  https://fast-foodfastapp.herokuapp.com/api/v1 - GET request
   - signup route -  https://fast-foodfastapp.herokuapp.com/api/v1/auth/signup - POST request
   - Login route -  https://fast-foodfastapp.herokuapp.com/api/v1/auth/login- POST request
   - placing an order -  https://fast-foodfastapp.herokuapp.com/api/v1/orders - POST request
   - order history of user -  https://fast-foodfastapp.herokuapp.com/api/v1/users/userId/orders    
 GET request
   - get all orders -  https://fast-foodfastapp.herokuapp.com/api/v1/orders - GET request (Accessible by just the admin)
   - get specific order -  https://fast-foodfastapp.herokuapp.com/api/v1/orders/orderId - GET request (Accessible by just the admin)
   - get specific order -  https://fast-foodfastapp.herokuapp.com/api/v1/orders/orderId - GET request (Accessible by just the admin)
   - update specific order -  https://fast-foodfastapp.herokuapp.com/api/v1/orders/orderId - PUT request (Accessible by just the admin)
   - see menu items -  https://fast-foodfastapp.herokuapp.com/api/v1/menu - GET request 
   - add menu items -  https://fast-foodfastapp.herokuapp.com/api/v1/menu - POST request (Accessible by just the admin)
 - After visiting these routes, you should be able to get the data and post more at a different time and the data will still be available.


### What are the relevant pivotal tracker stories?
The relevant pivotal story is:

Database connection  - https://www.pivotaltracker.com/story/show/160914128
